README.md

```diff
@@ -1,2 +1,173 @@
-# py-composite-action-lib
-PyCompositeActionLib is a Python library that streamlines creating and managing composite GitHub Actions. It offers reusable classes and scripts for controlling action inputs/outputs and accessing the GitHub API. 
+# PyCompositeActionLib
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [How to Use](#how-to-use)
+  - [Example: Control Action Inputs and Outputs](#example-control-action-inputs-and-outputs)
+  - [Example: Access GitHub API with GithubManager](#example-access-github-api-with-githubmanager)
+- [Features](#features)
+- [How to Use Local Version of Library for Development](#how-to-use-local-version-of-library-for-development)
+- [How to Run Tests](#how-to-run-tests)
+- [How to Release](#how-to-release)
+- [Contribution Guidelines](#contribution-guidelines)
+- [License Information](#license-information)
+- [Contact or Support Information](#contact-or-support-information)
+
+
+## Description
+**PyCompositeActionLib** is a Python library that streamlines creating and managing composite GitHub Actions. It offers reusable classes and scripts for controlling action inputs/outputs and accessing the GitHub API. 
```

Comment:
**Minor:**
Some long lines all around

Reply:
What is the pointed problem here?
- Too long line in md file? (final md representation does not care about new lines in example above)
- Reach better human reading?

Reply:
When rendered by `md` knowledgeable application it's not a problem. When i text editor, it's hard to read though. Afaik, in `md`, if not part of a bullet point, lines are usually line-broken to 80/120 characters.

Reply:
OK, I will apply it.